images/tomcat/tests/02-run.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+if [[ "${IMAGE_NAME}" == "" ]]; then
```

Comment:
This is more idiomatic:

`if [[ -z "${IMAGE_NAME}" ]]; then` 

(here and elsewhere)